package.json

```diff
@@ -11,6 +11,7 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "formik": "^2.2.6",
```

Comment:
Why not use materialUI?

https://material-ui.com/components/text-fields/

Reply:
Formik is more than just component library, it handles also validation, form submission etc.

Reply:
oh ok, nvm